python/requirements.txt

```diff
@@ -1,2 +1,11 @@
-# This file should list any python package dependencies.
-nose==1.3.3
\ No newline at end of file
+# Test
+nose>=1.3
+
+# Lint
+pep8>=1.7
+flake8>=2.5
+pyflakes>=1.2
+
+# Packaging
```

Comment:
I am not very familiar with the python ecosystem, but in java, bounds like this can be a cause for issue due to API breakages. Can you add some upper bounds as well?